src/plan/global.rs

```diff
@@ -272,17 +272,17 @@ pub trait Plan: Sized + 'static + Sync + Send {
 
     #[inline]
     fn stress_test_gc_required(&self) -> bool {
-        let pages = self.base().vm_map.get_cumulative_committed_pages();
```

Comment:
After this gets removed, `get_cumulative_committed_pages()` is no longer used. Probably either remove it, or leave a comment saying it was used for stress test GC, but now is unused.